packages/server/dataloader/LocalCache.ts

```diff
@@ -171,7 +171,7 @@ export default class LocalCache<T extends keyof CacheType> {
     Object.keys(this.cacheMap).forEach((key) => {
       if (!key.startsWith(`${table}:`)) return
       const doc = this.cacheMap[key]
-      Object.assign(doc, updater)
+      Object.assign(doc as any, updater)
```

Comment:
heh, yeah this file is very old & only used by 1 thing, it will absolutely go away when we move to pg so i didn't take too much time making it perfect. just want it to stop complaining until i can delete it 🙂